methods/main_methods.py

```diff
@@ -322,8 +322,12 @@ def show_classification_distributions(labels, data):
         labels (np.array): matrix of classification per pixel
         data (np.array): data
     """
+    unique_labels = list(np.unique(labels))
```

Comment:
Nicer way to figure out which labels to plot distributions for. The 0 label is avoided since it denotes outliers as well as the borders between grains (when using persistence watershed).